scipy/special/extra_special/_lambertw.h

```diff
@@ -27,7 +27,7 @@
 #include <limits>
 
 #include "_evalpoly.h"
-#include "error.h"
+#include "../error.h"
```

Comment:
@izaid, I'd missed this earlier, but I think we need this to make the include safe right?

Reply:
If that's where `error.h` is, then yes! But I think `error.h` should be moved into `extra_special/`